trackintel/io/postgis.py

```diff
@@ -106,7 +106,8 @@ def read_positionfixes_postgis(
     --------
     >>> pfs = ti.io.read_positionfixes_postgis("SELECT * FROM positionfixes", con, geom_col="geom")
     >>> pfs = ti.io.read_positionfixes_postgis("SELECT * FROM positionfixes", con, geom_col="geom",
-    ...                                        index_col="id", user_id="USER", tracked_at="time")
+    ...                                        index_col="id", user_id="USER",
```

Comment:
Shouldn't `user_id="USER"` be part of the `read_gpd_kws` dictionary?? `user_id` is not an keyword argument of the function, so I assume the code would not work like this

Reply:
good catch :)